pkg/sql/logictest/testdata/logic_test/alter_table

```diff
@@ -3551,21 +3551,13 @@ ADD CONSTRAINT "t_120017_pkey" PRIMARY KEY (a, c)
 
 subtest end
 
-subtest add_default_empty_array
+subtest regression_test_alter_table_add_constraint_unique
 
 statement ok
-CREATE TABLE t_114316 (i INT PRIMARY KEY)
+create table t_124546(a int);
 
-statement ok
```

Comment:
oh just noticed this. this is removing an existing test case. instead, you should add your new test without removing this one.